### PR TITLE
PL-1580 project structure updates

### PR DESCRIPTION
**Pull request by Gashaal**

### CODE REVIEW

File: src/frontend.md [hunk 1]
```diff
@@ -10,13 +10,14 @@ src/
...
         [avoid index.ts]
-        [other files, but component files .tsx, styled.tsx, local ui hooks, .test.ts, spec.ts, local static files should be placed in queries or services]
+        [other files, but component files .tsx, styled.tsx, stories.tsx, local ui hooks, .test.ts, spec.ts, local static files should be placed in queries or services]
         BankAccountTable.ts
         BankAccountTable.test.ts (unit tests)
         BankAccountTable.spec.ts (integration tests)
         BankAccountTableBadge.ts (child component)
         useBankAccountTableBadge.ts (ui hook used only in the current component)
         BankAccountTable.styled.ts
+        BankAccountTable.stories.tsx
```
Comment (Gashaal) [inline src/frontend.md:20]: found out in websome-accounting, seems useful.

File: src/frontend.md [hunk 2]
```diff
@@ -31,7 +32,8 @@ src/
         [other files should be placed in queries or services]
         [components for the page should be placed in components directory]
         BankAccountList.page.ts
-        BankAccountList.page.test.ts
+        BankAccountList.page.styled.tsx
+        BankAccountList.page.spec.ts (integration tests)
```
Comment (Gashaal) [inline src/frontend.md:36]: I think only integration tests must be here.

File: src/frontend.md [hunk 3]
```diff
@@ -10,13 +10,14 @@ src/
       BankAccountTable/
         [no nested directories]
         [avoid index.ts]
-        [other files, but component files .tsx, styled.tsx, local ui hooks, .test.ts, spec.ts, local static files should be placed in queries or services]
+        [other files, but component files .tsx, styled.tsx, stories.tsx, local ui hooks, .test.ts, spec.ts, local static files should be placed in queries or services]
         BankAccountTable.ts
         BankAccountTable.test.ts (unit tests)
         BankAccountTable.spec.ts (integration tests)
         BankAccountTableBadge.ts (child component)
         useBankAccountTableBadge.ts (ui hook used only in the current component)
```
Comment (Rendalf) [inline src/frontend.md:18]: could you also group hooks together? it would be easier to read

**Review by Rendalf — APPROVED** [review]

File: src/frontend.md [hunk 4]
```diff
@@ -31,7 +32,8 @@ src/
         [other files should be placed in queries or services]
         [components for the page should be placed in components directory]
         BankAccountList.page.ts
-        BankAccountList.page.test.ts
+        BankAccountList.page.styled.tsx
```
Comment (strashkov) [inline src/frontend.md:35]: `page.styled` files are deprecated
@divineforest fyi

**Pull request merged**